### PR TITLE
Update docs to mention the need for two adapters.

### LINKED ISSUE #40: Correct README
It is mentioned that you need two interfaces.Make it clear that you need two adapters.

### PR DESCRIPTION
**Pull request by jalcine**
Fixes #40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ From the victim's perspective, the attack makes use in three phases:
 
 ## Requirements
 * Kali Linux.
-* Two wireless network interfaces, one capable of injection.
+* Two wireless network adapters; one capable of injection. This allows you to connect and confirm the state of the target AP.
```
Comment (sophron) [inline README.md]: I don't think the last statement is correct. The one adapter is used for the injection, the other one for creating the fake AP. Although, I believe there's no need for an explanation in the 'Requirements' section. You can just remove the last sentence.

Comment (arthurbarros) [inline README.md]: @sophron +1

**Comment by jalcine** [comment]
Gotcha!

**Comment by sandman6578** [comment]
soprano sorry that i write in this issue pls show at my new issue i think its a bug

**Comment by sandman6578** [comment]
sophron*

**Comment by jalcine** [comment]
I could have sworn I committed that sooner; my bad for taking eons.

**Pull request merged**